src/fuchsia/mod.rs

```diff
@@ -357,6 +357,14 @@ s! {
         pub sin6_scope_id: u32,
     }
 
+    pub struct sockaddr_vm {
+        pub svm_family: sa_family_t,
+        pub svm_reserved: u16,
+        pub svm_port: crate::in_port_t,
+        pub svm_cid: u32,
```

Comment:
Usually we use the `c_*` types if that is how they are defined in the source.

```suggestion
        pub svm_reserved: c_short,
        pub svm_port: crate::in_port_t,
        pub svm_cid: c_int,
```

Do you have a link to relevant header for fuchsia?